.github/workflows/prestocpp-linux-build.yml

```diff
@@ -86,3 +86,34 @@ jobs:
         with:
           path: '${{ env.CCACHE_DIR }}'
           key: ccache-prestocpp-linux-build-engine
+
+      - name: Regenerate the protocol files
+        run: |
+          export PRESTO_HOME=$(pwd)
+          cd presto-native-execution/presto_cpp/presto_protocol
+          make presto_protocol
+
+      - name: Incrementally rebuild presto_server with the new protocol
+        run: |
+          cd presto-native-execution
+          cmake \
```

Comment:
Wonder if there is a way to reuse this command above and here. That will avoid updating in two places in the future.